### ISSUE TITLE
rocsparse-test spgemm_bsr.extra: Assertion `format == rocsparse_format_csc' failed.

### ISSUE THREAD

**Issue by littlewu2508** [ROCm/rocSPARSE#357]
### What is the expected behavior
- Test pass

### What actually happens
```
/clients/tests/../include/utility.hpp:484: rocsparse_local_spmat::rocsparse_local_spmat(int64_t, int64_t, int64_t, void *, void *, void *, rocsparse_indextype, rocsparse_indextype, rocsparse_index_base, rocsparse_datatype, rocsparse_format): Assertion `format == rocsparse_format_csc' failed.
```

By backtracing, it seems that  https://github.com/ROCmSoftwarePlatform/rocSPARSE/blob/rocm-5.7.1/clients/testings/testing_spgemm_bsr.cpp#L76 is calling https://github.com/ROCmSoftwarePlatform/rocSPARSE/blob/rocm-5.7.1/clients/include/utility.hpp#L455, but not https://github.com/ROCmSoftwarePlatform/rocSPARSE/blob/rocm-5.7.1/clients/include/utility.hpp#L520, which I cannot understand

The full build and test log (Gentoo rocSPARSE package) is attached:

[build.log.gz](https://github.com/ROCmSoftwarePlatform/rocSPARSE/files/13255961/build.log.gz)

### How to reproduce
- Run `./rocsparse-test --gtest_filter=*pre_checkin/spgemm_bsr.extra/i32_i32_f32_r_bad_arg*`

### Environment
| Hardware | description |
|-----|-----|
| GPU | AMD Instinct MI210 |
| CPU | AMD EPYC 7763 64-Core Processor |

| Software | version |
|-----|-----|
| ROCK | linux-6.1 (Debian 6.1.27-1) |
| ROCR |rocm-5.7.1 |
| HIP | rocm-5.7.1 |
| Library | rocm-5.7.1 |

**Comment by YvanMokwinski** [comment]
@littlewu2508 Thank you. This is indeed a bug. It will be fixed as soon as possible.

**Comment by ppanchad-amd** [comment]
@YvanMokwinski Is this fixed in the latest ROCm 6.1.2? Thanks

**Comment by jsandham** [comment]
Closing as this is fixed in 6.1.2.
